docs/menu.yml

```
@@ -37,13 +37,23 @@ catalog:
       - name: "Manual APIs"
         catalog:
           - name: "Tracing APIs"
-            path: "/en/setup/service-agent/java-agent/application-toolkit-trace"
+            catalog:
+              - name: "Add Trace Toolkit Dependencies"
+                path: "/en/setup/service-agent/java-agent/application-toolkit-dependency"
+              - name: "Use Native Tracer"
+                path: "/en/setup/service-agent/java-agent/application-toolkit-tracer"
+              - name: "Read Tracing Context"
+                path: "/en/setup/service-agent/java-agent/application-toolkit-trace-read-context"
+              - name: "Annotations"
+                path: "/en/setup/service-agent/java-agent/application-toolkit-trace-annotation"
+              - name: "Use Correlation Context"
+                path: "/en/setup/service-agent/java-agent/application-toolkit-trace-correlation-context"
+              - name: "Another Across Thread Solution"
```

```suggestion
              - name: "Across Thread Solution"
```

Another seems a strange word in doc menu